modules/mint-mode-with-removal-of-admin-credential.adoc

```diff
@@ -3,25 +3,19 @@
 // * installing/installing_aws/manually-creating-iam.adoc
```

Comment:
These changes have been reviewed in the other PR for 4.7 CCO work, but I copied them over since that one isn't merged yet.